app/src/test/java/AppTest.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class AppTest {
```

Comment:
Don't forget to test your methods OR something. You can always test happy path + empty path.